extensions/interactions/base_test.py

```diff
@@ -313,6 +337,28 @@ def test_default_interactions_are_valid(self):
             else:
                 self.assertIsNone(interaction.default_outcome_heading)
 
+            # Check that the rules for this interaction have object editor
+            # templates.
+            for rule_name, rule_dict in interaction.rules.iteritems():
+                rule_description = rule_dict['description']
+                param_list = rule_domain.get_param_list(rule_description)
+
+                for (_, param_obj_type) in param_list:
+                    # TODO(sll): Get rid of these special cases.
```

Comment:
Yes, the only reason this is here is the fuzzy-rules issue, I think (and perhaps nonnegativeint, since special-case logic in the frontend changes that to a dropdown of choices for multiple choice). @wxyxinyu -- if we end up removing the fuzzy rules in this commit, could you get rid of the relevant object types here and in extensions.objects too?